reconcile/openshift_resources_base.py

```diff
@@ -622,14 +670,22 @@ def fetch_desired_state(
         _log_lock.release()
         return
 
+    # _post_process_resources
+    # Some resources might need existent data in the cluster to populate all its fields.
```

Comment:
can you elaborate on this comment?

Reply:
At first, I wanted to sync the certificate data into the Route with the `_post_process_route` method. Finally, I removed the logic and let `cert-utils-operator` sync the data. I left the comment there because this could be extended in the future by other resource types. 

The main reason not to do the sync is that I want to cover the `saas` deployments too, and with those is not possible to get the secret data because they are not reconciled regularly like `openshift_resources`.